src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker_Generators.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this file is where all SG related code will go to keep things nicely separated out.